runtime/moonbase/src/governance/tracks.rs

```diff
@@ -0,0 +1,286 @@
+// Copyright 2022 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot. If not, see <http://www.gnu.org/licenses/>.
+
+//! Track configurations for governance.
+
+use super::*;
+use crate::currency::{KILOUNIT, SUPPLY_FACTOR, UNIT};
+
+const fn percent(x: i32) -> sp_runtime::FixedI64 {
```

Comment:
I think you will get the same precision and useful range out of a `Perbill` with only 32 bits.

Perbill: 32 bits, range: [0, 1] with 9 decimal points of precision
FixedI64: 64 bits, range [-9223372036.854775808, 9223372036.854775807]

Both have 9 points of decimal precision, the difference is in the integer portion, which we aren't using here.

Reply:
I don't see this explicitly used as storage, though. The point is irrelevant unless we're trying to save space somewhere.

Reply:
This helper function is used to form the inputs for all the curves via the curve helpers `make_linear` and `make_reciprocal`. These functions take `FixedI64` so that is the reason for it being returned here.